docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,392 @@
+# Getting Started
+This document is intended as a "getting started" guide. The audience are developers looking to modify or *publish* the packages to the Universe. This document is written in tutorial and walk-through format. The goal is to help you "get started". It does not go into great depth on some of the conceptual or inner details. This guide aims at making a user familiar with the concepts of what a Package is and what are the roles of Marathon and Universe in the package life cycle.
```

Comment:
Updated paragraph as per suggestion. Still kept the relevant information to make the user aware that this guide doesn't provide exhaustive information about all the configuration.